index.html

```diff
@@ -0,0 +1,201 @@
+          </div>
+        </section>
+        <section class="program" id="program">
+          <div class="row program-title container-fluid mx-auto">
+            <div class="col"></div>
+            <div class="col col- mt-3 pt-3"><h3>Main Program</h3></div>
+            <div class="col"></div>
+          </div>
+          <div class="row program-title container mx-auto">
+            <div class="col"></div>
+            <div class="col"><hr id="line"></div>
+            <div class="col"></div>
+          </div>
+          <div class="container-fluid">
+            <div class="row mt-3 cards mx-auto programs">
+              <div class=" col-md-2 card">
+                <img src="./images/speak.png" class="card-img-top" alt="...">
+                <div class="card-body">
+                  <h5 class="card-title">Lecture</h5>
+                  <p class="card-text">
+                    Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+                  </p>
+                </div>
+              </div>
+              <div class="col-md-2 card">
+                <img src="./images/exhibit.png" class="card-img-top" alt="...">
+                <div class="card-body">
+                  <h5 class="card-title">Exhibition</h5>
+                  <p class="card-text">
+                    Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+                  </p>
+                </div>
+              </div>
+              <div class="col-md-2 card">
+                <img src="./images/forum.png" class="card-img-top" alt="...">
+                <div class="card-body">
+                  <h5 class="card-title">Forum</h5>
+                  <p class="card-text">
+                    Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+                  </p>
+                </div>
+              </div>
+              <div class="col-md-2 card">
+                <img src="./images/wkshop.png" class="card-img-top" alt="...">
+                <div class="card-body">
+                  <h5 class="card-title">Workshop</h5>
+                  <p class="card-text">
+                    Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+                  </p>
+                </div>
+              </div>
+              <div class="col-md-2 card">
+                <img src="./images/ignite.png" class="card-img-top" alt="...">
+                <div class="card-body">
+                  <h5 class="card-title">Resources</h5>
+                  <p class="card-text">
+                    Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+                  </p>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="d-grid gap-2 col-8 mx-auto mt-5">
+            <button class="btn btn-primary" type="button">Book a Seat</button>
+          </div>
+          <div class=" d-grid gap-2 col-8 mx-auto mt-3 p-3">
+            <h3 class="lead">Guest speakers</h3>
+          </div>
```

Comment:
- [ ] Amazing work so far 👏 
- [ ] Book a seat is supposed to be moved at Guest speakers
- [ ] And guest speakers is moved insider the speakers section
- [ ] see screenshots below

`your tenplate`
![image](https://user-images.githubusercontent.com/66230306/192095058-864150bf-1d3e-43ce-af37-5abaa4ef6b5f.png)

`original design`
![image](https://user-images.githubusercontent.com/66230306/192095273-c207ba52-1efd-4775-8f93-840e0eba525e.png)